CMakeLists.txt

```diff
@@ -460,13 +460,47 @@ if(BUILD_TESTS)
   )
 
   add_e2e_test(
-    NAME test_suite
+    NAME rekey_tests
```

Comment:
rekey_test**~s~**?

Reply:
Sure, seems we've been a bit inconsistent on this. Are we happy with `recovery_test`, `rekey_test`, `governance_test`, `schema_test_...`, `election_test_...`, or does anyone think these are a deliberate plurality?

Reply:
I'm not sure the _test bit is even useful?

Reply:
I've turned `_tests` into `_test` for now. I'm also unconvinced by the `_test` suffix the more I think about it. Maybe its useful to distinguish labels, build targets, other concepts, from ctest tests? Let's rename broadly in a focussed PR if we're going to.